packages/media/src/getImageDimensions.js

```diff
@@ -26,7 +26,12 @@ import preloadImage from './preloadImage';
  * @return {Promise} Image dimensions object.
  */
 const getImageDimensions = async (src) => {
-  const img = await preloadImage(src).catch(() => {});
+  let img;
+  try {
+    img = await preloadImage(src);
+  } catch (error) {
+    //ignore
+  }
   return {
     width: img.naturalWidth,
```

Comment:
Now this would give an error if the `img` is `null`, no?

Reply:
https://github.com/google/web-stories-wp/pull/10068#discussion_r778719305